### PR TITLE
rewriting using zenpacklib

### PR DESCRIPTION
**Pull request by ssaporylo**

### CODE REVIEW

File: .gitignore~ [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*.py[co]
```
Comment (xroot88) [inline .gitignore~]: please, remove this file.

File: ZenPacks.community.Arista.egg-info/PKG-INFO [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 1.0
```
Comment (xroot88) [inline ZenPacks.community.Arista.egg-info/PKG-INFO]: remove this file.
should be in .gitignore

File: ZenPacks.community.Arista.egg-info/SOURCES.txt [hunk 3]
```diff
@@ -22,32 +22,19 @@ ZenPacks/community/Arista/AristaMemory.py
 ZenPacks/community/Arista/AristaMemory.pyc
```
Comment (xroot88) [inline ZenPacks.community.Arista.egg-info/SOURCES.txt]: should be in .gitingore
please, remove this file

File: ZenPacks/community/Arista/LICENSE.txt [hunk 4]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015, Arista Networks Inc.
```
Comment (xroot88) [inline ZenPacks/community/Arista/LICENSE.txt:1]: please, include this file

Comment (ssaporylo) [inline ZenPacks/community/Arista/LICENSE.txt:1]: fixed it

File: ZenPacks.community.Arista.egg-info/SOURCES.txt (hunk 3 above)
Comment (ssaporylo) [inline ZenPacks.community.Arista.egg-info/SOURCES.txt]: removed this file

File: ZenPacks.community.Arista.egg-info/PKG-INFO (hunk 2 above)
Comment (ssaporylo) [inline ZenPacks.community.Arista.egg-info/PKG-INFO]: removed this file

File: .gitignore~ (hunk 1 above)
Comment (ssaporylo) [inline .gitignore~]: removed this file

File: ZenPacks/community/Arista/reportable.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
+##############################################################################
```
Comment (xroot88) [inline ZenPacks/community/Arista/reportable.py:1]: Do you need this file?

File: ZenPacks/community/Arista/reportable.zcml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (xroot88) [inline ZenPacks/community/Arista/reportable.zcml:1]: Do you need this file?

**Comment by xroot88** [comment]
Please, remove all files that are should be in .gitignore, including all _pyc and *egg-info_ and others.
Add the Arista license file back in.

File: ZenPacks/community/Arista/reportable.zcml (hunk 6 above)
Comment (ssaporylo) [inline ZenPacks/community/Arista/reportable.zcml:1]: Yes

**Comment by xroot88** [comment]
looks good

**Pull request merged**